### DISCUSSION TITLE
Create Homebrew tap for automated installation and upgrading

### DISCUSSION THREAD

**Discussion by scmeek** [linearmouse/linearmouse#34] [Ideas]
Homebrew on macOS makes user management of applications like this so easy, including automated upgrading. It's also a godsend when managing "workflows" on different computers. It would be hugely beneficial to [add this to Homebrew](https://betterprogramming.pub/a-step-by-step-guide-to-create-homebrew-taps-from-github-repos-f33d3755ba74).

**Comment by lujjjh** [comment]
Thank you for the link. I'll put this into my backlog.

**Reply by scmeek** [reply]
Awesome! Thank you. I’m willing to help as well if you’re interested.

BTW, I love the app! Exactly what I was looking for and it works great. Thank you for writing it.

**Reply by lujjjh** [reply]
You're welcome! Your link is quite helpful. The only question that comes to mind is how 'brew upgrade' handles app upgrades while they are running. Will apps be restarted during the upgrade, or should I write some scripts to handle it?

**Reply by lujjjh** [reply]
[Homebrew Cask](https://github.com/Homebrew/homebrew-cask) might be a better choice for GUI apps. So I'm submitting LinearMouse to Homebrew Cask. Thank you for reminding me the package manager stuffs!

**Comment by lujjjh** [comment]
LinearMouse can now be installed via Homebrew:

```sh
$ brew install --cask linearmouse --no-quarantine
```
